cmd/ipfs-cluster-service/main.go

```diff
@@ -189,10 +189,10 @@ func main() {
 			Name:  "debug, d",
 			Usage: "enable full debug logging (very verbose)",
 		},
-		cli.StringFlag{
-			Name:  "loglevel, l",
-			Value: defaultLogLevel,
-			Usage: "set the loglevel for cluster components only [critical, error, warning, info, debug]",
+		cli.StringSliceFlag{
+			Name:   "loglevel, l",
+			EnvVar: "LOG_LEVEL",
```

Comment:
IPFS_CLUSTER_LOG_LEVEL

Cluster environment variables should have CLUSTER_ in them. `env | grep CLUSTER_` should show everything affecting cluster.